src/dao/BaseDAO.js

```diff
@@ -58,8 +55,9 @@ class BaseDAO<TEntity, TEntityMutator> {
   __setupHandler(
     handler: oHandler<TEntity>,
     queryOptions?: QueryOptions = {},
-    shouldSelectExpand: boolean = true
+    shouldSelectExpand: boolean = true,
   ): oHandler<TEntity> {
+    /* eslint-disable no-param-reassign */
```

Comment:
Do you still need this?

I actually think that `handler` isn't immutable so we probably don't need to reassign.